tools/devtool

```diff
@@ -562,7 +562,10 @@ ensure_ci_artifacts() {
 
     # Fetch all the artifacts so they are local
     say "Fetching CI artifacts from S3"
-    S3_URL=s3://spec.ccfc.min/firecracker-ci/v1.11/$(uname -m)
+    pushd $FC_ROOT_DIR/src/firecracker/src
+    FC_VERSION=$(cargo pkgid | cut -d# -f2 | rev | cut -d. -f2- | rev)
```

Comment:
But this is running using cargo from outside devctr, which we cannot guarantee will be available (for example in the CI metals). I think a `cmd_sh cargo ...` could work, maybe:

```sh
cmd_sh "cd src/firecracker; cargo pkgid | cut -d# -f2 | cut -d- -f1"
```

Also I think the `cut` is enough?

```
% echo 1.11.0 |cut -d- -f1
1.11.0
% echo 1.11.0-dev |cut -d- -f1
1.11.0
```

Reply:
Ah, missed that, thanks!

We need the second cut to transform `1.11.0` -> `1.11` because of how our artifacts bucket is named

Reply:
Ah right, would this work without the revs `cut -d. -f1-2`?

Reply:
O, neat, stackoverflow seemingly didnt know about that option :joy: fixed :)